### ISSUE TITLE
Error: Couldn't find a navigation object. Is your component inside a screen in a navigator?

### ISSUE THREAD

**Issue by danielkv** [nandorojo/expo-next-react-navigation#91]
Related to https://github.com/nandorojo/expo-next-react-navigation/issues/69

@nandorojo should I wrap the app with <NavigationContainer> even if I'm in web with next?

I'm having the same problem here, but even I'm using only useRouting(), it's returning the same error.

Ps: I'm currently not wrapping the Next app with <NavigationContainer>. Just using the standard Next navigation

**Comment by nandorojo** [comment]
Try fixing React Navigation to the version you're using with yarn resolutions.

**Comment by nandorojo** [comment]
Yeah, it would. Can you use yarn resolutions first to see if that solves it?

**Comment by danielkv** [comment]
> Yeah, it would. Can you use yarn resolutions first to see if that solves it?

I tried, but my knowleadge on resolutions is not that good. What I tried to do was:

```json
"resolutions": {
        "*/@react-navigation/native": "^5.9.3"
    }
```

**Comment by danielkv** [comment]
> The latest version of this library should work with v5 too actually. Try that

Oh, I'm gonna try that!
